repository-data/application/src/main/resources/hcm-content/content/documents/administration/national-indicator-library.yaml

```diff
@@ -0,0 +1,206 @@
+---
+/content/documents/administration/national-indicator-library:
+  /headers:
+    /headers[1]:
+      hippo:availability: []
+      hippostd:state: draft
+      hippostdpubwf:createdBy: admin
+      hippostdpubwf:creationDate: 2018-02-08T11:24:07.560Z
+      hippostdpubwf:lastModificationDate: 2018-02-08T11:24:07.560Z
+      hippostdpubwf:lastModifiedBy: admin
+      jcr:mixinTypes:
+      - mix:referenceable
+      jcr:primaryType: resourcebundle:resourcebundle
+      resourcebundle:descriptions:
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      - ''
+      resourcebundle:id: nationalindicatorlibrary.headers
+      resourcebundle:keys:
+      - publishedBy
```

Comment:
can you prefix all of these with `headers.`, we have a few different resorce files so it's good to be able to differentiate between then

Reply:
sorted